<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwpmSubmit</class>
 <widget class="QDialog" name="TwpmSubmit">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Submit your time to TeamWorkPM</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdTimeInfo">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="numDigits">
         <number>8</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="title">
         <string>Time to submit:</string>
        </property>
        <widget class="QLineEdit" name="txtHours">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>16</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtMinutes">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>16</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>18</y>
           <width>26</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>hour</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>18</y>
           <width>22</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBillable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Is Billable</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblOwner">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cmdSubmit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdAssignTask">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Assign Task</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>201</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="txtDescription"/>
      </item>
      <item>
       <widget class="QLabel" name="lblTask">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cmdAssignTask</sender>
   <signal>clicked()</signal>
   <receiver>TwpmSubmit</receiver>
   <slot>AssignTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>TwpmSubmit</receiver>
   <slot>CancelSubmission()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdSubmit</sender>
   <signal>clicked()</signal>
   <receiver>TwpmSubmit</receiver>
   <slot>SubmitToTwpm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtHours</sender>
   <signal>textEdited(QString)</signal>
   <receiver>TwpmSubmit</receiver>
   <slot>TimeUpdatedManually()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtMinutes</sender>
   <signal>textEdited(QString)</signal>
   <receiver>TwpmSubmit</receiver>
   <slot>TimeUpdatedManually()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CancelSubmission()</slot>
  <slot>AssignTask()</slot>
  <slot>SubmitToTwpm()</slot>
  <slot>ParseTimeAndUpdate()</slot>
  <slot>TimeUpdatedManually()</slot>
 </slots>
</ui>
